firebase-second-gen-preset/entry.ts

```diff
@@ -0,0 +1,20 @@
+import "#internal/nitro/virtual/polyfill";
+import { type HttpsOptions, onRequest } from "firebase-functions/v2/https";
+
+// auto imported
+// import { toNodeListener } from 'h3'
+
+const nitroApp = useNitroApp();
+
+const onRequestDefaults = {
+  memory: "1GiB",
+  // TODO: does this change anything?
+  invoker: "public",
```

Comment:
We need to enforce this otherwise the google cloud permissions aren't set properly, and they can only be changed with the gcloud cli (not the firebase cli or the dashboard)
We figured that out in the firebase-v2 pr preset

Reply:
Thank you, this is the kind of things that are hard to deal with in Firebase projects 😓 , it's the part that is not always document or easy to find

Reply:
What I find weird is that it wasn't necessary in gen 1?

Reply:
They changed the way permissions works at the gcloud level